django/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.12.4-slim
+
+WORKDIR /app
+
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED=1
+
+COPY conf/requirements.txt requirements.txt
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY conf/entrypoint.sh /entrypoint.sh
+RUN chmod a+x /entrypoint.sh
+
```

Comment:
Missing expose and volume declaration for documentation

Reply:
ok